bom/pom.xml

```diff
@@ -414,6 +414,11 @@
                 <artifactId>cxf-tools-common</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
```

Comment:
I believe we should add this dependency to `cxf-core` (which could be excluded on demand) to not break existing application if anyone is using `Compiler` class (since this is public API).